charts/prometheus-postgres-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "v0.15.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 6.2.0
+version: 6.2.1
```

As an enhancement/new feature, I'd suggest to make it a minor change.
```suggestion
version: 6.3.0
```

---

@zeritti Thank you for the review the change has been pushed